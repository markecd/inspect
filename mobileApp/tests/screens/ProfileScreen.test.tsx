import React from 'react';
import { render } from '@testing-library/react-native';
import { Text } from 'react-native';


const MockedProfileDetails = () => <Text>Mocked ProfileDetails</Text>;
const MockedCollectionDetails = () => <Text>Mocked CollectionDetails</Text>;
const MockedAchievementsDetails = () => <Text>Mocked AchievementsDetails</Text>;

jest.mock('@/components/Profile/ProfileDetails', () => () => MockedProfileDetails());
jest.mock('@/components/Profile/CollectionDetails', () => () => MockedCollectionDetails());
jest.mock('@/components/Profile/AchievementsDetails', () => () => MockedAchievementsDetails());


jest.mock('expo-router', () => ({
  useLocalSearchParams: () => ({}),
}));

import ProfileScreen from '@/app/profile/index';

describe('ProfileScreen', () => {
  it('ujema se s snapshotom', () => {
    const tree = render(<ProfileScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
