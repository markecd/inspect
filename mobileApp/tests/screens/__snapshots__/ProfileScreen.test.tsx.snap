// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileScreen ujema se s snapshotom 1`] = `
<RCTScrollView
  contentContainerStyle={
    {
      "backgroundColor": "#6C584C",
      "flex": 1,
    }
  }
>
  <View>
    <Text>
      Mocked ProfileDetails
    </Text>
    <Text>
      Mocked CollectionDetails
    </Text>
    <Text>
      Mocked AchievementsDetails
    </Text>
  </View>
</RCTScrollView>
`;
